website/docs/d/nsxt_segment_security_profile.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_segment_security_profile"
+sidebar_current: "docs-vcd-data-source-nsxt-segment-security-profile"
+description: |-
+  Provides a VMware Cloud Director NSX-T Segment Security Profile data source. This can be used to read NSX-T Segment Profile definitions.
+---
+
+# vcd\_nsxt\_segment\_security\_profile
+
+Provides a VMware Cloud Director NSX-T Segment Security Profile data source. This can be used to read NSX-T Segment Profile definitions.
+
+Supported in provider *v3.11+*.
+
+## Example Usage (Segment Security Profile)
+
+```hcl
+data "vcd_nsxt_manager" "nsxt" {
+  name = "nsxManager1"
+}
+
+data "vcd_nsxt_segment_security_profile" "first" {
+  name            = "segment-security-profile-0"
+  nsxt_manager_id = data.vcd_nsxt_manager.nsxt.id
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of Segment Profile
+* `nsxt_manager_id` - (Optional) Segment Profile search context. Use when searching by NSX-T manager.
```

Comment:
A general polish comment, but we should either use dots or not in the lists of the same page for tidiness.

Reply:
This is my flaw. We have it different across the code and it leaves me wondering. Will just have to stop using dots at everywhere I think :)